features-json/variable-fonts.json

```diff
@@ -317,7 +317,7 @@
     "1":"Works with Experimental Web Platform features enabled",
     "2":"Requires MacOS 10.12+ and the following about:config flags to be enabled:\r\n`layout.css.font-variations.enabled`,\r\n`gfx.downloadable_fonts.keep_variation_tables`",
     "3":"Requires MacOS 10.13+",
-    "4":"Does not support the `font-weight` and `font-stretch` properties, nor `format('truetype-variations')`"
```

Comment:
I believe this note also applies to Firefox, so you should probably change the one that applies to Chrome (& Opera) if it is incorrect.

Reply:
Done, I separated the notes, and tested additional format strings in FF, then updated FF accordingly.